meinberlin/apps/cms/models/storefronts.py

```diff
@@ -62,28 +63,55 @@ def project_url(self):
             return self.project.externalproject.url
         return self.project.get_absolute_url()
 
-    @cached_property
-    def district_project_count(self):
+    def get_district_project_count(self):
         projects = Project.objects.filter(
             Q(access=Access.PUBLIC) | Q(access=Access.SEMIPUBLIC),
             administrative_district=self.district,
             is_draft=False,
             is_archived=False,
         )
-        plans = Plan.objects.filter(district=self.district, status=0)
+        plans = Plan.objects.filter(
+            district=self.district,
+            status=0,
+            is_draft=False,
+        )
         active_project_count = plans.count()
         for project in projects:
             if project.active_phase or project.future_phases:
                 active_project_count += 1
         return active_project_count
 
+    def save(self, *args, **kwargs):
+        if not self.district_project_count:
+            self.district_project_count = self.get_district_project_count()
```

Comment:
Could we also do this with `default =` or does that not work with the district here?

Reply:
No, it doesn't work because we do not have the district when we would use the default.